### PR TITLE
removes Metajokes and ERP references from the arcade machine.

### PR DESCRIPTION
**Pull request by chaoko99**
Self explanitory really.

### CODE REVIEW

File: code/game/machinery/computer/arcade.dm [hunk 1]
```diff
@@ -99,10 +99,10 @@
 	var/name_part1
 	var/name_part2
 
-	name_action = pick("Defeat ", "Annihilate ", "Save ", "Strike ", "Stop ", "Destroy ", "Robust ", "Romance ", "Pwn ", "Own ", "Ban ")
```
Comment (Lohikar) [inline code/game/machinery/computer/arcade.dm]: These are not ERP references.

Comment (chaoko99) [reply code/game/machinery/computer/arcade.dm]: Reclarified the intent of the pull request.

Comment (MarinaGryphon) [reply code/game/machinery/computer/arcade.dm]: You didn't remove "robust". The others could pass as non-metajokes, as they're just part of shared gamer parlance that would make sense in the context of playing an arcade machine,  but robust is definitely one. Ban is one too, so "own" and "pwn" are the only ones of the four I think should stay.

File: code/game/machinery/computer/arcade.dm [hunk 2]
```diff
@@ -99,10 +99,10 @@
 	var/name_part1
 	var/name_part2
 
-	name_action = pick("Defeat ", "Annihilate ", "Save ", "Strike ", "Stop ", "Destroy ", "Robust ", "Romance ", "Pwn ", "Own ", "Ban ")
+	name_action = pick("Defeat ", "Annihilate ", "Save ", "Strike ", "Stop ", "Destroy ", "Robust ", "Romance ")
 
 	name_part1 = pick("the Automatic ", "Farmer ", "Lord ", "Professor ", "the Cuban ", "the Evil ", "the Dread King ", "the Space ", "Lord ", "the Great ", "Duke ", "General ")
-	name_part2 = pick("Melonoid", "Murdertron", "Sorcerer", "Ruin", "Jeff", "Ectoplasm", "Crushulon", "Uhangoid", "Vhakoid", "Peteoid", "slime", "Griefer", "ERPer", "Lizard Man", "Unicorn", "Bloopers")
+	name_part2 = pick("Melonoid", "Murdertron", "Sorcerer", "Ruin", "Jeff", "Ectoplasm", "Crushulon", "Uhangoid", "Vhakoid", "Peteoid", "slime", "Lizard Man", "Unicorn", "Bloopers")
```
Comment (MarinaGryphon) [inline code/game/machinery/computer/arcade.dm]: I believe Lizard Man may also be a /tg/ reference, but I'm not sure.

**Comment by skull132** [comment]
No, why?

I quite like the rare zany stuff that still exists in parts of old code. I don't really see anything wrong with it.

File: code/game/machinery/computer/arcade.dm (hunk 2 above)
Comment (chaoko99) [reply code/game/machinery/computer/arcade.dm]: I mean, lizard men are hardly a thing from TG exclusively.

Comment (chaoko99) [reply code/game/machinery/computer/arcade.dm]: Resolved the former, latter isn't a problem. Lizard people are not a construct of /TG/. They go back further than the greeks.

**Comment by OneOneThreeEight** [comment]
>Self explanatory really.

Not really. Explain why they're a problem?

**Comment by chaoko99** [comment]
Because they're dumb, and don't really make sense in world. Like at all. It's metahumor. I'm frankly rather annoyed by SS13's preference to incestuous memes rather than things that make sense.

**Comment by skull132** [comment]
Too many downvotes, and negative feedback from leadership as well. Going to close this tomorrow if there are no reasonable objections.

A part of SS13's charm is the zany shit and jokes in the game. Even on a heavy roleplay server, it is sometimes nice/funny to be reminded of those things. It's not like the jokes are obstructive to gameplay, either.

**Comment by chaoko99** [comment]
Call it what you want. This is still a MRP server.

Half-Jokes aside: I just think it's a fairly ridiculous thing to close one thing (FLB nerfs) because it's memey, but at the same time, object to something that is removing meme bullshit.

**Comment by skull132** [comment]
You only read half my post.

> A part of SS13's charm is the zany shit and jokes in the game. Even on a heavy roleplay server, it is sometimes nice/funny to be reminded of those things. **It's not like the jokes are obstructive to gameplay, either.**

**Comment by chaoko99** [comment]
Okay but who would ever make a game that would be sold to a workplace about kicking the asses of ERPers. It's really not funny, not zany, just incestuous and bad.

**Comment by skull132** [comment]
People much alike the people who:
* Would build a station without stairs/emergency stairs.
* Issue weapons to miners as tools.
* Let scientists produce live weapons and ammo without being under guard.
* Host two different, dangerous engines in one complex on the same powernet without actually reliable backups.

I could go on. SS13 is zany by nature, surely you cannot argue this?

**Comment by chaoko99** [comment]
Fine, whatever. Someone needs to redo arcade machines anyways.

**Pull request closed without merging**